_includes/sidebar.html

```diff
@@ -20,9 +20,9 @@
       <li><a href="https://codelabs.developers.google.com/codelabs/flutter/index.html#0">Building beautiful UIs - Codelab</a></li>
       <li><a href="/tutorials/layout/">Build layouts - Tutorial</a></li>
       <li><a href="/tutorials/interactive/">Add interactivity - Tutorial</a></li>
-      <li><a href="/web-analogs/">HTML/CSS patterns</a></li>
-      <li><a href="/flutter-for-android/">Flutter for Android Developers</a></li>
-      <li><a href="/flutter-for-react-native/">Flutter for React Native Developers</a></li>
+      <li><a href="/web-analogs/">Flutter for Web devs</a></li>
```

Comment:
This may be a bridge too far (by web devs != HTML/CSS coders). No strong opinion, though.